### ISSUE TITLE
Improper end column with unclosed quote

### ISSUE THREAD

**Issue by Arcanemagus** [stylelint/stylelint#718]
With this CSS file:

```css
.s["] {
}
/* "" */
```

`stylelint` reports the column of the unclosed quote on line 1 as being 18, which is far past the end of the line. It seems like it is counting all characters between the first quote and the next one as part of the line? When the comment with the `"` is removed the proper value is returned. If that comment is brought to the same line as the initial `"` the issue disappears as well.

Tested with `stylelint` v4.2.0, based on a report initially filed here: https://github.com/AtomLinter/linter-stylelint/issues/79.
![image](https://cloud.githubusercontent.com/assets/427137/12932639/88718672-cf38-11e5-87c8-94a7376ada7c.png)

**Comment by Arcanemagus** [comment]
Actually, should this be filed over on [postcss](https://github.com/postcss/postcss)?

**Comment by davidtheclark** [comment]
Ultimately, yes. But I'll do that after investigating a bit. I just found a similar bug so might look into it a bit before filing an issue with PostCSS -- to see if it's related to the other thing I already found or not.

**Comment by Arcanemagus** [comment]
Okay, let me know if you need any more information :wink:.

**Comment by davidtheclark** [comment]
Not related to the other bug I found with positioning. I'll file an issue that just copies your report. Thanks!
